docs/registry/_index.md

```diff
@@ -27,18 +27,32 @@ aliases:
 menuindent: true
 ---
 
+The [Viam registry](https://app.viam.com/registry) is the central place where you can browse:
+
+- [ML models](/ml/) to deploy with machine applications like [computer vision](/ml/vision/)
+- {{< glossary_tooltip term_id="modular-resource" text="modular resources" >}} that add capabilities to your machine beyond what is built-in to `viam-server`
+
+### ML models
+
+Viam provides the ability to train, upload, and deploy ML models within the platform.
+See [Machine Learning](/ml/) for more information.
+
+The Viam registry hosts pretrained ML models that users have made public, which you can use to deploy classifiers or detectors for your use case onto your robot instead of training your own.
```

Comment:
I don't know that pretrained has much meaning here. You can't do any additional training, so they're jsut trained I guess? Sorry, I know we've already looked at this. 
```suggestion
The Viam registry hosts ML models that users have made public, which you can use to deploy classifiers or detectors for your use case onto your robot instead of training your own.
```

Reply:
No worries sounds good

Reply:
Can I still use trained? I feel it makes it more clear

Reply:
yep sounds good